### PR TITLE
added autoscaling group

### PR DESCRIPTION
**Pull request by geoffrey1330**

### CODE REVIEW

File: level2/autoscaling.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  vpc_zone_identifier  = data.terraform_remote_state.networking.outputs.public-subnet_id
+
+  load_balancers    = ["${aws_elb.app-lb.name}"]
+  health_check_type = "ELB"
+
+  max_size = 5
+  min_size = 2
+
+  tag {
+    key                 = "name"
```
Comment (vladimirmukhin) [inline level2/autoscaling.tf]:
```suggestion
    key                 = "Name"
```

File: level2/autoscaling.tf [hunk 2]
```diff
@@ -0,0 +1,16 @@
+resource "aws_autoscaling_group" "app-asg" {
+  launch_configuration = aws_launch_configuration.web_config.id
+  vpc_zone_identifier  = data.terraform_remote_state.networking.outputs.public-subnet_id
+
+  load_balancers    = ["${aws_elb.app-lb.name}"]
+  health_check_type = "ELB"
```
Comment (vladimirmukhin) [inline level2/autoscaling.tf]:
```suggestion
```

File: level2/autoscaling.tf [hunk 3]
```diff
@@ -0,0 +1,16 @@
+resource "aws_autoscaling_group" "app-asg" {
+  launch_configuration = aws_launch_configuration.web_config.id
+  vpc_zone_identifier  = data.terraform_remote_state.networking.outputs.public-subnet_id
+
+  load_balancers    = ["${aws_elb.app-lb.name}"]
```
Comment (vladimirmukhin) [inline level2/autoscaling.tf]:
```suggestion
  load_balancers    = [aws_elb.app-lb.name]
```

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

File: level2/autoscaling.tf (hunk 2 above)
Comment (geoffrey1330) [reply level2/autoscaling.tf]: Should i remove it at all??

Comment (vladimirmukhin) [reply level2/autoscaling.tf]: Yes, it's an optional field. By default it is EC2 which is a way to go

Comment (geoffrey1330) [reply level2/autoscaling.tf]: Alright

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**